src/core/Browser.js

```diff
@@ -51,7 +44,7 @@ const gecko3d = 'MozPerspective' in style;
 
 // @property any3d: Boolean
 // `true` for all browsers supporting CSS transforms.
-const any3d = !window.L_DISABLE_3D && (webkit3d || gecko3d) && !opera12 && !phantom;
+const any3d = !window.L_DISABLE_3D && (webkit3d || gecko3d) && !phantom;

```

Comment:
Out of topic but is phantom still a valid browser?

Reply:
#8622

Reply:
Already have a PR for removing that as well! See #8622.